dev-tools/mage/dockerbuilder.go

```diff
@@ -182,13 +182,26 @@ func (b *dockerBuilder) expandDockerfile(templatesDir string, data map[string]in
 
 func (b *dockerBuilder) dockerBuild() (string, error) {
 	tag := fmt.Sprintf("%s:%s", b.imageName, b.Version)
+	aliasTag := ""
```

Comment:
@jsoriano I was not able to easily execute this locally (create the elastic-agent docker image) with one single command. Do you know the build steps?

Reply:
`mage package` in the `elastic-agent` directory should generate all the packages, including the docker images. There is no command to generate only the docker images.
You may want to select the platforms with the `PLATFORM` environment variable, if not it will generate all the packages for all the platforms, what will probably take a while.